ArtOfIllusion/src/artofillusion/SplineMeshEditorWindow.java

```diff
@@ -758,8 +758,7 @@ void subdivideCommand()
     param = new double [usize+usplitcount][vsize][numParam];
     for (i = 0; i < param.length; i++)
       for (j = 0; j < param[i].length; j++)
-        for (int k = 0; k < param[i][j].length; k++)
```

Comment:
Just repeated the test with two texture parameters mapped. The results are reversed. For a fairly small subdivide, (subdividing an 8x16 spline mesh) the arrayCopy version is approximately 14% slower. (mean of 5 runs) Larger subdivides are worse, but there seem to be other bottlenecks. The times for this run are all over the place.

Of note: I added a counter to see how many times the inner loop in question gets called: 81 loops through this bit of code to subdivide a 4X8 mesh.